src/client/src/components/ListControls.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+
+import { Button } from "monday-ui-react-core";
+import "monday-ui-react-core/dist/main.css";
+
+import PropTypes from "prop-types";
+
+ListControls.propTypes = {
+  itemToCreate: PropTypes.func.isRequired,
+};
+function ListControls({ loading, setLoading,itemToCreate }) {
+  const [inputText, setInputText] = useState("");
+
+  const inputTextHandler = () => {
```

Comment:
since you're using it as a callback you need to wrap it with useCallback.
also I would rename it to an action name, `handleInputText` instead of `inputTextHandler`

Reply:
```
const inputTextHandler = useCallback(() => {
...
}, [inputText, itemToCreate, setInputText])
```